src/styles/navigation.css

```
@@ -0,0 +1,417 @@
+  outline: 2px solid rgba(255, 255, 255, 0.5);
+  outline-offset: 2px;
+}
+
+.hamburger-menu-item:focus,
+.settings-menu-item:focus {
+  outline: 2px solid #0891b2;
+  outline-offset: -2px;
+}
+
+/* High contrast mode support */
+@media (prefers-trast: high) {
```

Biome (2.1.2)

[error] Don't use unknown media feature names.

Unexpected unknown media feature name.
You should use media feature names defined in the CSS Specifications.

(lint/correctness/noUnknownMediaFeatureName)